src/lib/utils.js

```diff
@@ -0,0 +1,9 @@
+export function makeLogger(prefix) {
+  if (process.env.NODE_ENV === 'development') {
+    return (...args) => {
+      console.log(`SnoozeTabs (${prefix})`, ...args);  // eslint-disable-line no-console
```

Comment:
It did, but doesn't really matter... just arbitrary logging chatter, not parsed by anything except eyeballs